test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -2172,6 +2172,37 @@ public void CreateDescriptors_BuildsDescriptorsFromSimpleTypes()
                 TagHelperAttributeDescriptorComparer.Default);
         }
 
+        public static TheoryData<string, string> HtmlConversionData
```

Comment:
@dougbu makes it easier to change when modifying existing tests. It's also what we do most elsewhere in the Razor code base.